doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -38,3 +38,119 @@ further documented in the `.mplot3d.axes3d.Axes3D.view_init` API.
 
 .. plot:: gallery/mplot3d/view_planes_3d.py
    :align: center
+
+
+Rotation with mouse
+===================
+
+3D plots can be reoriented by dragging the mouse.
+There are various ways to accomplish this; the style of mouse rotation
+can be specified by setting ``rcParams.axes3d.mouserotationstyle``, see
+:doc:`/users/explain/customizing`.
+
+Originally (with ``mouserotationstyle: azel``), the 2D mouse position
+corresponded directly to azimuth and elevation; this is also how it is done
+in `MATLAB <https://www.mathworks.com/help/matlab/ref/view.html>`_.
+This approach works fine for polar plots, where the *z* axis is special;
+however, it leads to a kind of 'gimbal lock' when looking down the *z* axis:
+the plot reacts differently to mouse movement, dependent on the particular
+orientation at hand. Also, 'roll' cannot be controlled.
+
+As an alternative, there are various mouse rotation styles where the mouse
+manipulates a 'trackball'. In its simplest form (``mouserotationstyle: trackball``),
+the trackball rotates around an in-plane axis perpendicular to the mouse motion
+(it is as if there is a plate laying on the trackball; the plate itself is fixed
+in orientation, but you can drag the plate with the mouse, thus rotating the ball).
+This is more natural to work with than the ``azel`` style; however,
+the plot cannot be easily rotated around the viewing direction - one has to
+drag the mouse in circles with a handedness opposite to the desired rotation.
+
+A different variety of trackball rotates along the shortest arc on the virtual
+sphere (``mouserotationstyle: arcball``); it is a variation on Ken Shoemake's
+ARCBALL [Shoemake1992]_. Rotating around the viewing direction is straightforward
+with it. Shoemake's original arcball is also available
+(``mouserotationstyle: Shoemake``); it is free of hysteresis, i.e.,
+returning mouse to the original position returns the figure to its original
+orientation, the rotation is independent of the details of the path the mouse
+took. However, Shoemake's arcball rotates at twice the angular rate of the
+mouse movement (it is quite noticeable, especially when adjusting roll).
```

Comment:
This is adjustable with `rcParams.axes3d.trackballsize`, right?

Reply:
Well, yes and and no. For the ``trackball`` style, it is a wholehearted 'yes!', the ``trackballsize`` just controls the angular rate. But for the other styles, the size also controls where the edge of the arcball is (how far from the center), that is, where the transition is from mostly controlling tilt (left/right and up/down) in the center, to mostly controlling roll near the edge. Moreover, you cannot make the 'roll' discrepancy (twice the angular rate) go away by adjusting the ``trackballsize``: no matter what size the trackball is, dragging the mouse around it will make the plot follow at the right or at double the speed (depending of the style).

Reply:
A 3D plot in the docs of the control surface might be useful!

Reply:
> A 3D plot in the docs of the control surface might be useful!

It probably would, but I'll focus on fixing the other issues in the review for now, if that's ok.